templates/openshift/rbac/role.yaml

```diff
@@ -7,6 +8,18 @@ rules:
       - ""
     resources:
       - configmaps
```

Comment:
We would have to add all the resoruces and apigroups that the backupjob needs to access. That would make it similar to https://github.com/integr8ly/backup-container-image/pull/5/files#diff-1857ec47aa300369db1c2fa542f40842R6

Reply:
This is the current rbac file that's in master. I took out secrets since this won't be needed here, i'm just not sure what is the configmaps being used for so I haven't taken it out. I think this can be updated/removed with your PR.

Reply:
Right, on master there was only secrets and configmaps in that cluster role. With my PR i've changed that to include all the resources that I need to be able to read. The problem is, if I add all those resources, then `backupjob-secret-reader` becomes redundant because the cluster role that we have gives the sa access to that secret anyway.

I'm going to experiment a bit more with this.

Reply:
@pb82 Saw that change when I rebased. Yes that's true, the role/rolebinding here will just get overridden by the clusterrole/clusterrolebinding on master. Feel free to close this one if you find another solution.